src/components/slider/_slider.scss

```diff
@@ -0,0 +1,110 @@
+//-----------------------------
+// Slider
+//-----------------------------
+
+@import '../../globals/scss/colors';
+@import '../../globals/scss/helper-mixins';
+@import '../../globals/scss/layer';
+@import '../../globals/scss/typography';
+@import '../../globals/scss/layout';
+@import '../../globals/scss/import-once';
+@import '../form/form';
+
+@include exports('slider') {
+  .bx--slider-container {
+    max-width: rem(600px);
+    display: flex;
+    align-items: center;
+    user-select: none;
+  }
+
+  .bx--slider {
+    position: relative;
+    width: 100%;
+    margin: 0 rem(12px);
+  }
+
+  .bx--slider--disabled {
+    opacity: .5;
+    & > * {
+      cursor: not-allowed !important; // fix
+    }
+  }
+
+  .bx--slider__range-label {
+    @include font-size('14');
+    color: $text-02;
+
+    &:last-of-type {
+      margin-right: 1rem;
+    }
+  }
+
+  .bx--slider__track {
+    position: absolute;
+    width: 100%;
+    height: rem(4px);
+    background: $ui-05;
+    cursor: pointer;
+    transform: translate(0%, -50%);
+  }
+
+  .bx--slider__filled-track {
+    position: absolute;
+    width: 100%;
+    height: rem(4px);
+    background: $brand-01;
+    transform-origin: left;
+    pointer-events: none;
+    transform: translate(0%, -50%);
+  }
+
+  .bx--slider__thumb {
+    position: absolute;
+    height: rem(24px);
+    width: rem(24px);
+    background: $brand-01;
+    border-radius: 50%;
+    top: 0;
+    transform: translate(-50%, -50%);
+    transition: transform 100ms $bx--standard-easing, background 100ms $bx--standard-easing;
```

Comment:
weirdest thing - it only happens on my monitor and not every time 😛  So ignore my comment!